redis.conf

```diff
@@ -346,6 +346,7 @@ pidfile /var/run/redis_6379.pid
 # verbose (many rarely useful info, but not a mess like the debug level)
 # notice (moderately verbose, what you want in production probably)
 # warning (only very important / critical messages are logged)
+# nothing (nothing is logged, that is close the log)
```

Comment:
```suggestion
# nothing (nothing is logged)
```
I kind of get what you are saying about "closing the log", but I don't think it's obvious what it means. If we want more information, I would clarify why it's useful.